go/oasis-test-runner/scenario/e2e/e2e.go

```diff
@@ -27,37 +28,50 @@ type e2eImpl struct {
 	net    *oasis.Network
 	name   string
 	logger *logging.Logger
-
-	// nodeBinary is the path to oasis-node executable.
-	nodeBinary string
+	flags  *flag.FlagSet
 }
 
 func newE2eImpl(name string) *e2eImpl {
-	return &e2eImpl{
-		name:       "e2e/" + name,
-		logger:     logging.GetLogger("scenario/e2e/" + name),
-		nodeBinary: "oasis-node",
+	// Empty scenario name is used for registering global parameters only.
+	fullName := "e2e"
+	if name != "" {
+		fullName += "/" + name
+	}
+
+	sc := &e2eImpl{
+		name:   fullName,
+		logger: logging.GetLogger("scenario/e2e/" + name),
```

Comment:
if `name == ""` then this will be `"scenario/e2e/"`? maybe use `"scenario/fullName"`